src/containers/AdyenInitialPayment/AdyenInitialPayment.tsx

```diff
@@ -20,17 +21,22 @@ type Props = {
 };
 
 export default function AdyenInitialPayment({ setUpdatingOrder, type, setPaymentError, paymentSuccessUrl, orderId }: Props) {
+  const accountController = getModule(AccountController);
+  const checkoutController = getModule(CheckoutController);
+
   const [session, setSession] = useState<AdyenPaymentSession>();
 
-  const { sandbox } = useClientIntegration();
+  const getIntegration = useConfigStore((s) => s.getIntegration);
+  const { useSandbox } = getIntegration();
```

Comment:
This is used in a couple places. Maybe it makes sense to make a function for it? Or just shorten it to a single line?
```suggestion
  const { useSandbox } = useConfigStore((s) => s.getIntegration());
```